src/ecrc-frontend/src/App.js

```diff
@@ -31,9 +32,15 @@ export default function App() {
   };
 
   const page = {
+    org,
     setOrg,
     pageLayout
   };
 
-  return <div>{!org.orgNm && <OrgValidation page={page} />}</div>;
+  return (
+    <div>
```

Comment:
if you'd like this entire section could be 😸 
```
{(org.orgNm) ? <OrgVerification page={page} /> : <OrgValidation page={page} />}
```

Reply:
This is just taking the place of a router for now to render pages, it could be squished but if it were to continue I would be adding something like {!org.whitelist && <Redirect />} so that if it came back from the api but wasnt a whitelisted org code it would send them to the redirect page to go to the old app. And so on for various pages, and would eventually become gross.

Reply:
yeah this was just a stylistic suggestion - it will be handled through routing eventually anyways so thats fine